From 229c785af9f190e2a8e3a7af39b133739f1a418b Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Thu, 21 Mar 2024 11:22:49 +0200
Subject: [PATCH] [FSW-4100] fs1: add dummy common file

Some Makefiles expect content in the arch/common directory.
for FS1 we avoid putting code in the common dir. To satisfy compilation
needs few dummy files are created in the common dir.

Change-Id: I0232c9a610ef94360d8352e06b9ef2503d6eb7ac
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/fs1/common/Makefile b/arch/fs1/common/Makefile
new file mode 100644
index 0000000..7b6fbcc
--- /dev/null
+++ b/arch/fs1/common/Makefile
@@ -0,0 +1,3 @@
+include $(HL_DEV_ARCH)/Makefile
+
+# this Makefile is for backwards compatibility, please use ARC Makefile
diff --git a/arch/fs1/common/common.c b/arch/fs1/common/common.c
new file mode 100644
index 0000000..00c0dfa
--- /dev/null
+++ b/arch/fs1/common/common.c
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier:     GPL-2.0+
+/*
+ * (C) Copyright 2024 HabanaLabs, Ltd.
+ */
+
+/* dummy file for compilation, please don't put code here */
diff --git a/arch/fs1/common/pcie_phy_fw.mk b/arch/fs1/common/pcie_phy_fw.mk
new file mode 100644
index 0000000..a3010e8
--- /dev/null
+++ b/arch/fs1/common/pcie_phy_fw.mk
@@ -0,0 +1,3 @@
+include $(HL_DEV_ARCH)/pcie_phy_fw.mk
+
+# this makefile is for backwards compatibility, please use ARC Makefile
